hedera-node/test-clients/src/main/java/com/hedera/services/bdd/spec/transactions/token/HapiTokenUnlock.java

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.bdd.spec.transactions.token;
+
+import static com.hederahashgraph.api.proto.java.ResponseCodeEnum.SUCCESS;
+
+import com.google.common.base.MoreObjects;
+import com.google.protobuf.ByteString;
+import com.hedera.services.bdd.spec.HapiSpec;
+import com.hedera.services.bdd.spec.transactions.HapiTxnOp;
+import com.hedera.services.bdd.spec.transactions.TxnUtils;
+import com.hedera.services.bdd.suites.HapiSuite;
+import com.hederahashgraph.api.proto.java.*;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.function.Consumer;
+import java.util.function.Function;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+public class HapiTokenUnlock extends HapiTxnOp<HapiTokenUnlock> {
+    static final Logger log = LogManager.getLogger(HapiTokenUnlock.class);
+
+    private final String account;
+    private final String token;
+    private long amount;
+    private final List<Long> serialNumbers;
+    private final SubType subType;
+    private ByteString alias = ByteString.EMPTY;
+
+    private boolean rememberingNothing = false;
+
+    @Override
+    public HederaFunctionality type() {
+        return HederaFunctionality.TokenUnlockUserAssets;
```

Comment:
Hi @netopyr,  In order to change the name of functionality from "TokenUnlockUserAssets" to "TokenUnlock", I think we have to make changes in "Hedera-protobuf". If someone make changes in protobuf then I can implement at my end. Otherwise, Can I incorporate this change along with my next PR (Delete Partition Token functionality). Thanks

Reply:
Hi @thomas-swirlds-labs, I have pushed changes. Can you please check it now